laghos_rom.hpp

```diff
@@ -223,7 +229,13 @@ class ROM_Sampler
     const int rank;
     double energyFraction;
 
+    const int parameter;
```

Comment:
`parameter` is vague. Maybe `parameterID` would be more descriptive. Then the local variable `parameterID` in the constructor could be removed.